serve/mlc_serve/api/handler.py

```diff
@@ -192,6 +196,7 @@ def create_stream_response(
                     finish_reason=seq.finish_reason.value
                     if seq.finish_reason is not None
                     else None,
+                    logprob_info=seq.logprob_info[0] if seq.logprob_info else None
```

Comment:
Please make it clear what the indexing by `0` means.

Reply:
It seems better to introduce a class instead of using a tuple, which is confusing. I'll refactor this part with a new class.